src/main/java/me/xmrvizzy/skyblocker/utils/ItemUtils.java

```diff
@@ -15,6 +19,56 @@
 public class ItemUtils {
     private final static Pattern WHITESPACES = Pattern.compile("^\\s*$");
 
+    public record Durability(int current, int max) {
+    }
+
+    @Nullable
+    public static Durability getDurability(ItemStack stack) {
+        if (!Utils.isOnSkyblock() || !SkyblockerConfigManager.get().locations.dwarvenMines.enableDrillFuel || stack.isEmpty()) {
+            return null;
+        }
+
+        NbtCompound tag = stack.getNbt();
+        if (tag == null || !tag.contains("ExtraAttributes")) {
+            return null;
+        }
+
+        NbtCompound extraAttributes = tag.getCompound("ExtraAttributes");
+        if (!extraAttributes.contains("drill_fuel") && !extraAttributes.getString("id").equals("PICKONIMBUS")) {
+            return null;
+        }
+
+        int current = 0;
+        int max = 0;
+        String clearFormatting = "";
```

Comment:
```suggestion
        String clearFormatting;
```

Reply:
Yeah good point, I just copied the existing code but will do.